qiita_pet/templates/list_analyses.html

```diff
@@ -2,9 +2,15 @@
 
 {% block head %}
 <script type="text/javascript">
+
+
   $(document).ready(function(){
     {% if analyses %}
       $('#no-analyses').hide()
+      $('#analyses-table').dataTable({
+        "order": [[ 3, "desc" ]]
+      });
+      $('#analyses-table').DataTable();
```

Comment:
Funny enough it doesn't as we are not changing that value but I also realized that it will be better to sort by 2 (timestamp), changing.